buildSrc/src/main/kotlin/com/datadog/gradle/plugin/apisurface/GenerateApiSurfaceTask.kt

```diff
@@ -0,0 +1,69 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-2020 Datadog, Inc.
+ */
+
+package com.datadog.gradle.plugin.apisurface
+
+import java.io.File
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.InputDirectory
+import org.gradle.api.tasks.OutputFile
+import org.gradle.api.tasks.TaskAction
+
+open class GenerateApiSurfaceTask : DefaultTask() {
```

Comment:
Is there a way to add unit tests for Gradle plugins' code? If yes, maybe it's worth adding at least integration test for this.

Reply:
Testing task is not trivial, and most of the work is delegated to the KotlinFileVisitor anyway.